src/main/java/appeng/integration/modules/waila/part/P2PStateWailaDataProvider.java

```diff
@@ -21,12 +21,16 @@
 
 import java.util.List;
 
+import appeng.api.util.AEColor;
+import appeng.util.Platform;
 import com.google.common.collect.Iterators;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Wrong order for 'com.google.common.collect.Iterators' import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)